x-pack/docs/en/security/authentication/user-profile.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[user-profile]]
+=== User profiles
+
+beta::[]
+
+User profiles provide persistent and stable representations of end-users.
+Persistent means it exists without the need for the user to be online.
+Stable means it has an unique identifier that does not change through
+the lifetime of a deployment and can be used to refer to the associated
+user.
+Being able to uniquely refer to users is a critical function that underpins
+important features like personalisation and collaboration in {kib}.
+
+==== How user profile works
+
+In the {stack}, a user is required to authenticate each request. Each user is
```

Comment:
Thanks for rewriting this section. I adopted with a minor edit: Instead of saying a profile is associated with "one or more users". I changed to it to say "is associated to a user". I think it is confusing at this point to hint that profile support the same user from different realms. We do talk about it in the following section after taking about the basic. So I'll keep this section a bit simpler.